jdaviz/configs/default/plugins/markers/markers.py

```diff
@@ -1,15 +1,18 @@
 import numpy as np
 from astropy import units as u
-from traitlets import Bool, observe

```

Comment:
we follow this pattern everywhere else, so would be nice to keep rather than calling `traitlet.` everywhere in just this one file.  So maybe keep this but use `traitlets.` for other calls that don't make sense to explicitly import?